src/components/MoreJobs.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <ion-card>
+    <ion-card-header>
+      <ion-card-title>{{ $t("More jobs") }}</ion-card-title>
+    </ion-card-header>
+    <ion-list>
+      <ion-item v-for="job in jobs" :key="job.jobId" @click="viewJobConfiguration(job)" detail button>
+        <ion-label class="ion-text-wrap">{{ job.jobName }}</ion-label>
+        <ion-label slot="end">{{ getTemporalExpression(job.enumTypeId) }}</ion-label>
+      </ion-item>
+    </ion-list>
+  </ion-card>
+</template>
+
+<script lang="ts">
+import {
+  IonCard,
+  IonCardHeader,
+  IonCardTitle,
+  IonItem,
+  IonLabel,
+  IonList
+} from '@ionic/vue';
+import { defineComponent } from 'vue';
+import { mapGetters, useStore } from 'vuex';
+import emitter from '@/event-bus';
+import { useRouter } from 'vue-router'
+
+export default defineComponent({
+  name: 'MoreJobs',
+  components: {
+    IonCard,
+    IonCardHeader,
+    IonCardTitle,
+    IonItem,
+    IonLabel,
+    IonList
+  },
+  data() {
+    return {
+      title: '',
+    }
+  },
+  props: ["jobs", "jobEnums"],
+  computed: {
+    ...mapGetters({
+      getJobStatus: 'job/getJobStatus',
+      getJob: 'job/getJob',
+      currentShopifyConfig: 'user/getCurrentShopifyConfig',
+      currentEComStore: 'user/getCurrentEComStore',
+      getTemporalExpr: 'job/getTemporalExpr',
+      getEnumName: 'job/getEnumName',
+    })
+  },
+  methods: {
+    async viewJobConfiguration(job: any) {
+      job.enumName = job.enumName || this.getEnumName(job.systemJobEnumId)
+      emitter.emit('showJobConfigurationForMoreJobs', {jobId: job.jobId, jobTitle: this.title, jobStatus: job.status, job: job});
```

Comment:
The title will always be empty for more jobs as we are not assigning a value to it. Refer to pipeline page to define job title using enum.

Reply:
Ok sir